PublicationsRouter/v2/oai-pmh/README.md

```diff
@@ -0,0 +1,28 @@
+# PubRouter OAI-PMH endpoint

```

Comment:
Throughout...

OAI-PMH tends to use terminology of `verb` to generally describe things like: `Identify`, `ListIdentifers`, ...  do can you change it to that in situations highlighted here (note that usages of "request" elsewhere in the doc are OK).

![image](https://user-images.githubusercontent.com/13220848/40483358-9ab86248-5f4f-11e8-9636-268380c92566.png)

![image](https://user-images.githubusercontent.com/13220848/40483371-a66089d6-5f4f-11e8-94d1-3998a547caf8.png)

![image](https://user-images.githubusercontent.com/13220848/40483391-ba613f8e-5f4f-11e8-844a-3a59c1cbf16a.png)

![image](https://user-images.githubusercontent.com/13220848/40483408-cdb7ef74-5f4f-11e8-9c5a-401a6c9c3316.png)
![image](https://user-images.githubusercontent.com/13220848/40483428-db711672-5f4f-11e8-9ca2-fcbbfc87601d.png)

![image](https://user-images.githubusercontent.com/13220848/40483437-e425a9f4-5f4f-11e8-841d-9d296535bbc8.png)